src/state/reducer.js

```diff
@@ -332,10 +332,9 @@ export default (state: State = idle, action: Action): State => {
     }
 
     invariant(
-      target.isEnabled === isEnabled,
-      `Trying to set droppable isEnabled to ${String(
-        isEnabled,
-      )} but it is already ${String(isEnabled)}`,
+      target.isEnabled !== isEnabled,
```

Comment:
this invariant check was the wrong way around.

We need to throw if they are the same.

Reply:
We need these values to be different - otherwise it is an error